### DISCUSSION TITLE
"Cart ROM-Reader" 10 Year Anniversary 🎂

### DISCUSSION THREAD

**Discussion by sanni** [sanni/cartreader#762] [General Chat]
Ten years ago today MichlK released [Cart_ROM_Reader.ino](https://web.archive.org/web/20141104175020/http://forum.arduino.cc/index.php?topic=158974.0):

![cartromreader](https://user-images.githubusercontent.com/19169494/230583795-49cc7092-fda5-47e8-a298-aa0aff03ed1b.jpg)

I'm gonna quote a part from his readme:

````
To give something back to the Arduino & SNES community, I hereby would like to 
share my results in the spirit of Arduino, quoting the Arduino homepage:
"Build it, hack it and share it. Because Arduino is you!"

I will most probably not spend any more time on this project because I have 
reached my goal to read the ROM of all my SNES game cartridges and I have 
learned much on my way there. If you would like to contribute, there are many 
things left to be done, e.g. reading / writing of the Save RAM of the game
cartridges. Please share your results as well!
````

So many thanks to MichlK for getting the ball rolling and encouraging others to build it, hack it and share it, which we did. 🔧🔨😊 
The evolution the Cart Reader took from there can be found in the [Wiki](https://github.com/sanni/cartreader/wiki/About).

Happy Birthday Cart ROM-Reader 🎂

**Comment by tamanegitaro** [comment]
Happy Birthday

**Comment by PsyK0p4T** [comment]
10 years... and where are we now ! Happy birthday, and huge thanks to all contributors across the years :)

**Comment by Chomemel** [comment]
Happy birthday CART-ROM Reader. Thanks to everyone who have been working one these projects that brought us asuch a wonderful device.
